Lookuptable/testPages/home/home_page.py

```diff
@@ -0,0 +1,82 @@
+from selenium.webdriver.common.by import By
+
+from common_utilities.selenium.base_page import BasePage
+from Lookuptable.userInputs.user_inputs import UserData
+
+""""Contains test page elements and functions related to the Homepage of Commcare"""
+
+
+class HomePage(BasePage):
```

Comment:
Again Redundant! [This ](https://github.com/dimagi/dimagi-qa/blob/master/HQSmokeTests/testPages/home/home_page.py) should be referenced instead, wherever its used.

Reply:
Changes made. Commit code for reference : fccca641344bce18df6c10d99660adae9e687091